### ISSUE TITLE
Hide "Offline" Playlist when there are no downloads

### ISSUE THREAD

**Issue by blindman81** [Malopieds/InnerTune#197]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

Hide the "offline" playlist in the library when there are no downloads.

### Why do you want this feature?

I stream music, so having that big offline playlist is just a waste of space.

**Comment by Malopieds** [comment]
Yeah that could be something nice but the thing is the way it is implemented for now, we need to check for all songs if it is downloaded so that would be too costly. Maybe one day this system will change and this will become a nice idea !
